ITechArt_project/settings.py

```diff
@@ -37,6 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'MainApp',
```

Comment:
It's not a bug, but quite different from common codestyle in Django. All applications, folders etc should be in lower case and snake_case like `main_app` here